Stanford/JumpstartEngineering/Install/Layouts/DeptHomepage.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @file
+ * Configure PICAL homepage layouts.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Layouts;
+/**
+ *
+ */
+class DeptHomepage extends \AbstractInstallTask {
+
+  /**
+   * Configure Department homepage layouts.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
```

Comment:
Hey @sherakama,
Thanks for the comments.  

I was thinking that lines 34-48 were disabling the other contexts, (e.g.  $context_status[$context_name] = TRUE;)
and were necessary. Will you please clarify?